sources/packages/web/src/components/students/applicationTracker/InProgress.vue

```diff
@@ -0,0 +1,296 @@
+        order to provide their information for your application.</span
+      ></template
+    ></application-status-tracker-banner
+  >
+
+  <application-status-tracker-banner
+    label="Waiting for additional information from another parent"
+    icon="fa:fas fa-clock"
+    icon-color="secondary"
+    v-if="applicationDetails?.parent2Info?.waiting"
+    ><template #content
+      ><span
+        >We are waiting for
+        <strong>supporting information from another parent.</strong> Please
+        check your email to confirm that you have received a message from us.
+        This email will include important details and links that your parent
+        will need in order to provide their information for your
+        application.</span
+      ></template
+    ></application-status-tracker-banner
+  >
+
+  <application-status-tracker-banner
+    label="Waiting for additional information from your partner"
+    icon="fa:fas fa-clock"
+    icon-color="secondary"
+    v-if="applicationDetails?.partnerInfo?.waiting"
+    ><template #content
+      ><span
+        >We are waiting for
+        <strong>supporting information from your partner.</strong> Please check
+        your email to confirm that you have received a message from us. This
+        email will include important details and links that your partner will
+        need in order to provide their information for your application.</span
+      ></template
+    ></application-status-tracker-banner
+  >
+
+  <application-status-tracker-banner
+    label="Waiting for your parent's income verification"
+    icon="fa:fas fa-clock"
+    icon-color="secondary"
+    content="The Canada Revenue Agency (CRA) is verifying your parent's income."
+    v-if="!!applicationDetails?.parent1IncomeVerificationStatus?.waiting"
+  />
+
+  <application-status-tracker-banner
+    label="Waiting for your other parent's income verification"
+    icon="fa:fas fa-clock"
+    icon-color="secondary"
+    content="The Canada Revenue Agency (CRA) is verifying your other parent's income."
+    v-if="applicationDetails?.parent2IncomeVerificationStatus?.waiting"
+  />
+
+  <application-status-tracker-banner
+    label="Waiting for your partner's income verification"
+    icon="fa:fas fa-clock"
+    icon-color="secondary"
+    content="The Canada Revenue Agency (CRA) is verifying your income."
+    v-if="applicationDetails?.partnerIncomeVerificationStatus?.waiting"
+  />
+
+  <application-status-tracker-banner
+    label="Waiting for StudentAid BC to review your application"
+    icon="fa:fas fa-clock"
+    icon-color="secondary"
+    content="StudentAid BC is currently reviewing your application and the 
+    additional information you provided for 1 or more of the questions."
+    v-if="
+      applicationDetails?.exceptionStatus === ApplicationExceptionStatus.Pending
+    "
+  />
+
+  <!-- Success cards. -->
+  <application-status-tracker-banner
```

Comment:
We can have a component that will hold the `waiting` and `success` card states and decide which one to show based on a condition. `icon-color` and `icon` would be defined internally by the component and the success/wait/error could be grouped.

Reply:
As we talked, the order in which the components are displayed would be better following the states (error, waiting, success). As also talked with @hellolynn-tbtb we will be keeping the component as it is right now.